src/cgame/cg_event.cpp

```diff
@@ -473,52 +472,44 @@ static void CG_Obituary( entityState_t *ent )
 				break;
 
 			case MOD_LEVEL1_CLAW:
-				message = G_( "%s%s ^*was sliced by %s%s%s^*'s %s" );
-				messageAssisted = G_( "%s%s ^*was sliced by %s%s%s^*'s %s^*; %s%s%s^* assisted" );
-				attackerClass = PCL_ALIEN_LEVEL1;
+				message = G_( "%s%s ^*was sliced by %s%s%s^*'s claws" );
+				messageAssisted = G_( "%s%s ^*was sliced by %s%s%s^*'s claws; %s%s%s^* assisted" );
```

Comment:
Shouldn't we maintain consistency with current behaviour? In which case, this should say:

`"%s%s ^*was sliced by %s%s%s^*'s Mantis"`